pkg/web/apps/apps_repository.go

```diff
@@ -14,5 +14,12 @@ type Repository interface {
 	DeleteAppByAppID(appId string) error
 	CreateApp(id, appId, name string) error
 	GetAppByAppID(appID string) (*models.App, error)
+	GetActiveAppByAppID(appID string) (*models.App, error)
```

Comment:
We need thee GetAppByID and the GetActiveAppByAppID.
We also need re-check this points @austincunningham